packages/website/src/pages/related-tools.tsx

```diff
@@ -79,6 +79,11 @@ const tools = [
     description: 'Gitmoji extension for Raycast',
     link: 'https://www.raycast.com/ricoberger/gitmoji',
   },
+  {
+    name: 'genmoji',
+    description: 'Gitmoji commit message generation using AI',
+    link: 'https://genmoji.xyz',
```

Comment:
Hey! We usually link the GitHub repo here, can you change this to:

```suggestion
    link: 'https://github.com/segersniels/genmoji',
```

Reply:
Done 👍